client/wfprev-war/src/main/angular/src/app/components/edit-project/activities/activities.component.ts

```diff
@@ -739,18 +741,41 @@ export class ActivitiesComponent implements OnChanges, CanComponentDeactivate {
           this.isNewActivityBeingAdded = false;
           return;
         }
-        // Delete from the service call if it's a saved fiscal activity
-        this.projectService.deleteActivity(this.projectGuid, this.fiscalGuid, activityGuid)
+        this.attachmentService.getActivityAttachments(this.projectGuid, this.fiscalGuid, activityGuid)
```

Comment:
ProjectFilesComponent is already the child component of activities component, so you should be able to viewChild on it to access the fileAttachment, without calling api.